desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/PathBrowser.tsx

```diff
@@ -97,125 +91,125 @@ const PathBrowser = ({
   };
 
   if (breadcrumbs) {
+    if (isEditMode) {
+      return (
+        <div ref={breadcrumbInputRef}>
+          <Input
+            prefix={showIcon ? icons[fileSystem] : <span />}
+            defaultValue={filePath}
+            onPressEnter={customPath => {
+              onFilepathChange((customPath.target as HTMLInputElement).value);
```

Comment:
customPath is event right? may be event.target is better, because i was thinking where is customPath coming from.